docs/website/docs/intro.md

```diff
@@ -26,42 +32,59 @@ pipeline = dlt.pipeline(
 # Grab some player data from Chess.com API
 data = []
 for player in ['magnuscarlsen', 'rpragchess']:
-  response = requests.get(f'https://api.chess.com/pub/player/{player}')
-  response.raise_for_status()
-  data.append(response.json())
+    response = requests.get(f'https://api.chess.com/pub/player/{player}')
+    response.raise_for_status()
+    data.append(response.json())
 # Extract, normalize, and load the data
 pipeline.run(data, table_name='player')
 ```
 
-[How it works](./reference/explainers/how-dlt-works.md): `dlt` extracts data from a [source](./general-usage/glossary.md#source), inspects its structure to create a [schema](general-usage/glossary.md#schema), structures, normalizes and verifies the data,
-and then loads the data into a [destination](./general-usage/glossary.md#destination).
+[How it works](reference/explainers/how-dlt-works.md): `dlt` extracts data from a
+[source](general-usage/glossary.md#source), inspects its structure to create a
+[schema](general-usage/glossary.md#schema), structures, normalizes and verifies the data, and then
+loads the data into a [destination](general-usage/glossary.md#destination).
 
 ## Who is `dlt` for?
 
 ### Why does the data team love dlt?
```

Comment:
I think you might have missed some other files. Would be nice to have the same formatting in all of them :)